tests/functional/UseCase/Bucket/GrantRevokeBucketAccessToReadOnlyRoleTest.php

```diff
@@ -58,7 +58,7 @@ public function testRegisterNonExistExchanger(): void
         $parsedName = AnalyticsHubServiceClient::parseName($createdListing->getName());
 
         $handler = new GrantBucketAccessToReadOnlyRoleHandler($this->clientManager);
-        $handler->setLogger($this->log);
+        $handler->setInternalLogger($this->log);
```

Comment:
@zajca este to chybalo na jednom mieste daj sem prosim 👍

Reply:
ok

Reply:
Este z nejakeho dovodu to po rebase chybalo tu 193b7911cbf832d9ad6c9f50d98a38a3b825ff59 tak mi len daj prosim palecek @zajca a mergnem to dik

Reply:
Jo protože sů to nové testy co přidal jirka :heavy_check_mark: